src/BSLightingShaderMaterialPBR.cpp

```diff
@@ -0,0 +1,314 @@
+#include "BSLightingShaderMaterialPBR.h"
+
+#include "State.h"
+
+BSLightingShaderMaterialPBR::~BSLightingShaderMaterialPBR()
+{}
+
+BSLightingShaderMaterialPBR* BSLightingShaderMaterialPBR::Make()
+{
+	return new BSLightingShaderMaterialPBR;
+}
+
+RE::BSShaderMaterial* BSLightingShaderMaterialPBR::Create()
+{
+	return Make();
+}
+
+void BSLightingShaderMaterialPBR::CopyMembers(RE::BSShaderMaterial* that)
+{
+	BSLightingShaderMaterialBase::CopyMembers(that);
+
+	auto* pbrThat = static_cast<BSLightingShaderMaterialPBR*>(that);
+
+	loadedWithFeature = pbrThat->loadedWithFeature;
+	pbrFlags = pbrThat->pbrFlags;
+	coatRoughness = pbrThat->coatRoughness;
+	coatSpecularLevel = pbrThat->coatSpecularLevel;
+	fuzzColor = pbrThat->fuzzColor;
+	fuzzWeight = pbrThat->fuzzWeight;
+	projectedMaterialBaseColorScale = pbrThat->projectedMaterialBaseColorScale;
+	projectedMaterialRoughness = pbrThat->projectedMaterialRoughness;
+	projectedMaterialSpecularLevel = pbrThat->projectedMaterialSpecularLevel;
+
+	if (rmaosTexture != pbrThat->rmaosTexture) {
+		rmaosTexture = pbrThat->rmaosTexture;
```

Comment:
Why not just assign without any condition?

Reply:
True. Again no idea why I left it so

Reply:
Removed